typeset/mjnode.js

```diff
@@ -87,14 +87,16 @@ const convertMathML = async (log, mathEntries/* [{xml: string, fontSize: number}
   const promises = mathEntries.map(({xml: mathSource, fontSize}) => {
     const id = done + index
     index++
-    return mjAPI.typeset({
+    const typesetConfig = {
       math: mathSource,
-      format: mathSource[0] !== '\\' ? 'MathML' : 'inline-TeX', // "inline-TeX", "TeX", "MathML"
+      format: mathSource.startsWith('<') ? 'MathML' : 'inline-TeX', // "inline-TeX", "TeX", "MathML"
```

Comment:
Is there possibility that inline-TeX equation will start with `<`? If so then maybe we should check for `<math` or add even more complex regexp for detecting if mathSource includes html elements?

Reply:
Or we could pass this information along with `xml` and `fontSize to the `mathMap` since in `converter.js` we have all information that we need to determine `format`

Reply:
I had a look and the tag depends on the html, so for example, you may have `<m:math` or `<math` depending on the namespaces are set up.  Still, it's possible to use a regular expression to detect it...  (I'm kind of lazy so I would prefer to not change the code so much..)

Reply:
Ok I changed it to `mathSource.match('<([^:]+:)?math')`

Reply:
`^` is needed at the beginning since `mc^{2} <math` will be also matched in this case.
I'm thinking now that passing `format` to `mathMap` could be actually more accurate

Reply:
I added `^` now.  I was thinking about `re.match` in python and `^` is implicit there :joy: Thanks for checking.